eos/form-factors/k-lcdas.cc

```diff
@@ -269,9 +270,11 @@ namespace eos
     AntiKaonLCDAs::phi(const double & u, const double & mu) const
     {
         // Gegenbauer polynomials C_n^(3/2)
-        const double x = 2.0 * u - 1.0, x2 = x * x;
-        const double c1 = 3.0 * x;
-        const double c2 = (15.0 * x2 - 3.0) / 2.0;
+        const GegenbauerPolynomial g1(1, 3.0 / 2.0);
```

Comment:
Hm. I would move this line into the constructor and provide all needed Gegenbauer polynomials as a member variable, e.g.
```C++
class PionLCDAs :
   ...
{
    const GegenbauerPolynomial gp_3o2_1;
    
    
    double
    PionLCDAs
};

PionLCDAs::PionLCDAs(...) :
    ...
    gp_3o2_1(1, 3.0 / 2.0),
    ...
{
    ...
}

double
PionLCDAs::phi(const double & u, const double & mu) const
{
    ...
    const double c1 = gp_3o2_1.evaluate(x), c2 = gp_3o2_2.evaluate(x);
    ...
}
```
In this way, the computationally expensive part (determining all coefficients) is done only once, while the fast part is called whenever is ``phi`` is called.